### PR TITLE
Refactor non-inbox queries (speed up and correct sort)

### PR DESCRIPTION
**Pull request by mgcam**
A visible speed-up for the display of the QC complete wells.

### CODE REVIEW

**Review by nerdstrike — CHANGES_REQUESTED** [review]
All good, a couple of typos and quibbles.

File: lang_qc/db/helper/wells.py [hunk 1]
```diff
@@ -45,3 +75,103 @@ def wells2paged_wells(self, wells: List[PacBioWell]) -> PacBioPagedWells:
...
+    def _add_tracking_info(self, wells: List[PacBioWell]):
+
+        # Will use the run name and well label to retrieve relevant rows from mlwh.
+        # In future, when available in mlwh for all records, product IDs can be used.
+
+        for well in wells:
+            # One query for all or query per well? The latter for now to avoid the need
+            # to match the records later. Should be fast enough for small-ish pages, we
+            # query on a unique key.
+            db_well = (
```
[the remark this answers is not in the document]
Comment (mgcam) [reply lang_qc/db/helper/wells.py:159]: I think brackets are added bt `black`. I'll see whether this interference can be stopped.

[the remark this answers is not in the document]
Comment (mgcam) [reply lang_qc/db/helper/wells.py:159]: No, cannot do anything.

[the remark this answers is not in the document]
Comment (nerdstrike) [reply lang_qc/db/helper/wells.py:159]: Not to worry then...

**Pull request merged**